config/settings/base.py

```diff
@@ -520,7 +520,7 @@
 # By default this is empty, you can find options in ``adserver.analyzer.backends``
 ADSERVER_ANALYZER_BACKEND = env.list(
     "ADSERVER_ANALYZER_BACKEND",
-    default=[],
+    default=["adserver.analyzer.backends.naive.NaiveKeywordAnalyzerBackend"],
```

Comment:
Not 100% sure we want this, but the requirements for the naive implementation are pretty small, basically just beautifulsoup

Reply:
By putting it in the default, the analyzer is going to be defaulted to on even for people who take our OSS code. I don't think we want that.. I say we keep the `default=[]` but put this as the default in `testing.py` so the naive analyzer gets tested. Thoughts?